readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://YuraMisura/layout_moyo-header)
```

Comment:
These links are broken - pay attention how to do it correctly next time: https://mate-academy.github.io/layout_task-guideline/#update-codedemo-linkcode-and-codetest-report-linkcode

Reply:
ok. I will be more attentive next time and i have updeted Demo and Report links